src/utils/logs.ts

```diff
@@ -2,9 +2,6 @@
 export function serverLog(val: any, callee?: string) {
   if (process.env.NODE_ENV === 'development') {
     const isCallee = callee ? `Called by ${callee}` : '';
-
-    return Array.isArray(val)
-      ? console.log(...val, isCallee)
-      : console.log(val, isCallee);
+    return console.log(...(Array.isArray(val) ? val : [val]), isCallee);
```

Comment:
Won't this make our app slow

Reply:
1. This will make only our development env slow because of the development check
2. And by slower if you mean 0.000001ms slower than yes it will make the development server slow

Reply:
That 0.000001ms will be be minutes for your pc in development

Reply:
And I am okay with that

Reply:
But I am not with that slow work, I need things fast even it is 0.000001ms